docs/guides/cloud/branch-review.mdx

```diff
@@ -0,0 +1,207 @@
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/test2.png"
+  alt="Test Comparison View"
+/>
+
+## Github Pull Request Comments
+
+The Cypress Cloud Github Integration offers detailed test information via [PR comments](/guides/cloud/integrations/source-control/github#Pull-request-comments). Follow the "Review all test suite changes" link to load the Branch Review between the source and base branches.
+
+<DocsImage
+  src="/img/guides/cloud/integrations/github/pr-comment-fail.jpg"
+  alt="Cypress GitHub App PR"
+/>
+
+## Troubleshooting
+
+### Note on Available Data
+
+Branch Review is a powerful tool to compare two branches with recorded runs to Cypress Cloud. There are factors that can impact what is available to review between a source and base branch.
+
+Most importanly, unless a PR is open (not closed or merged), the branch review is not guaranteed to show you the latest run recorded for the branch. When a PR transitions into a closed/merged state, it does not consider any subsequent commits pushed to the branch as part of the PR itself. Be sure that the latest CI job has been triggered or you may see the _closest_ commit and not the one that was _just_ merged.
+
+Additionally, whether or not a branch at a particular commit has a run recorded to Cypress Cloud will affect what is displayed. The table below captures scenario across both branches:
+
+|              Base Branch              |             Source Branch             | What Branch Review Displays                         |
+| :-----------------------------------: | :-----------------------------------: | --------------------------------------------------- |
+| <Badge type="success">has run</Badge> | <Badge type="success">has run</Badge> | Comparison data using both found runs               |
+| <Badge type="success">has run</Badge> |  <Badge type="danger">no run</Badge>  | Comparison data using last run on source branch     |
+|  <Badge type="danger">no run</Badge>  | <Badge type="success">has run</Badge> | Non-comparison data with found source run           |
+|  <Badge type="danger">no run</Badge>  |  <Badge type="danger">no run</Badge>  | Non-comparison data using last run on source branch |
+
+## Best Practices
+
+### Grouping Test Runs
+
+Cypress Cloud allows for [grouping recorded tests](/guides/cloud/smart-orchestration/parallelization#Grouping-test-runs) together under a single run. This means multiple `cypress run` calls can be labeled and associated to a single run in Cypress Cloud.
+
+It's important to avoid recording random sets of tests between branches. Recording multiple runs per commit without grouping can cause issues, as Branch Review relies on the _latest_ runs. Therefore, recording one run per commit and utilizing grouping to ensure accurate comparisons during branch reviews is recommended to improve effectiveness.
```

Comment:
Not even just "random", but any disjoint sets. Recording multiple runs on a commit will screw things up 100%. Any scenario where multiple runs happen on a single commit, grouping should just be used to group them into a single run. (CI retries are the only case where obviously they should not be grouped, and we will just pull the latest run which should work for that use case).

Reply:
I think the concept is there but getting muddy as-is. I'll try a less is more approach here to see if we feel the scenario is covered.